### PR TITLE
Describing env variables in readme file 

### PR DESCRIPTION
**Pull request by pacostas**
This PR describes all env variables that consumer and producer supports

### CODE REVIEW

**Comment by helio-frota** [comment]
So according to the https://github.com/nodeshift-starters/reactive-example/pull/99 this PR is the doc related right

File: README.md [hunk 1]
```diff
@@ -107,8 +139,9 @@ package. In the code you'll see a call to:
 
 which will get the connection info from the bindings if it is available and
 return it in the format required by the Kafka client.
-                                                                                        122,1         Bot
+122,1 Bot
```
Comment (mhdawson) [inline README.md]: What is this line?

Comment (pacostas) [reply README.md]: I think its not necessary. I'm removing it.

Comment (pacostas) [inline README.md]:
```suggestion
```

**Pull request merged**